src/DependencyInjection/ConfigResolver/HttpCacheConfigParser.php

```diff
@@ -37,6 +37,10 @@ public function addSemanticConfig(NodeBuilder $nodeBuilder)
                     ->scalarNode('varnish_invalidate_token')
                         ->info('Optional: Varnish Invalidation token for purge')
                         ->defaultNull()
+                    ->end()
+                    ->scalarNode('varnish_bulk_purge')
+                        ->info('If purge should happen in bulk or not (if on varnish-module 0.9.x you should disable this)')
+                        ->defaultTrue()
```

Comment:
If we add this setting, maybe it should default to `false`? and be set to `true` in 2.5.0 meta repo?

Reply:
maybe, and that is one of the reason I'm leaning towards option B from PR description now.